src/app/tests/suites/certification/Test_TC_LTIME_1_1_Simulated.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
+
+name: 108.1.1. [TC-LTIME-1.1] Global Attributes [DUT as Client]
+
+PICS:
+    - LTIME.C
+
+config:
+    nodeId: 0x12344321
+    cluster: "Time Format Localization"
+    endpoint: 0
+
+tests:
+    - label: "Wait for the device to be commissioned"
+      cluster: "DelayCommands"
+      command: "WaitForCommissioning"
+
+    - label: "DUT read (0xFFFD) ClusterRevision attribute"
+      PICS: LTIME.C.Afffd
+      wait: "readAttribute"
+      attribute: "ClusterRevision"
```

Comment:
Add 'response' tag to verify the result

Reply:
@yufengwangca In simulated tests response check for attribute read is not implemented in YAML.